crates/notary/server/Cargo.toml

```diff
@@ -44,3 +47,4 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true, features = ["env-filter"] }
 uuid = { workspace = true, features = ["v4", "fast-rng"] }
 ws_stream_tungstenite = { workspace = true, features = ["tokio_io"] }
+sgx-dcap-ql-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", tag = "DCAP_1.21", optional = true }
```

Comment:
```suggestion

sgx-dcap-ql-rs = { git = "https://github.com/intel/SGXDataCenterAttestationPrimitives", tag = "DCAP_1.21", optional = true }
```

Reply:
> nice find! can we leave a comment here on why we choose this repo instead of the other listed in https://github.com/intel/linux-sgx?tab=readme-ov-file#introduction? just for future reference :)

Which other libs? The other things mentioned at your link are a kernel module for older versions of Linux,  a patched openssl impl, a k8s gadget, etc

Reply:
> > nice find! can we leave a comment here on why we choose this repo instead of the other listed in https://github.com/intel/linux-sgx?tab=readme-ov-file#introduction? just for future reference :)
> 
> Which other libs? The other things mentioned at your link are a kernel module for older versions of Linux, a patched openssl impl, a k8s gadget, etc

no referring to any specific one, but just thought a comment in the `Cargo.toml` explaining why we picked this `SGXDataCenterAttestationPrimitives` repo will be informative